app/src/main/res/xml/file_paths.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<paths>
+    <external-path
+        name="picoverImages"
+        path="Android/data/com.intive.picover/files/Pictures" />
```

Comment:
why not to use `ContextCompat.getExternalFilesDirs`? And I think we should consider if we want to share our photos with other apps (then probably we should save them in some public directory)

Reply:
I'm using `Context::getExternalFilesDirs` but the paths are needed. About sharing the photos with other apps – we will discuss it later, it's only a draft to start feature 😄

Reply:
I can't see where you're using `picoverImages`, could you explain what this path is needed for?

Reply:
Yeah, it is possible to simplify it a little bit using `external-files-path` instead of `external-path`. Have a look now!